FraihmworkHealthAndIntegrityApi.yaml

```
@@ -25,6 +25,9 @@ tags:
   
 - name: UUID Control
   description: Functions for helping manage UUIDs for components and faults
+  
+- name: Blocklist
+  description: Blocklist management apis  
```

To match other tag descriptions.
```suggestion
  description: Blocklist management api endpoints
```